python/ec2_sg_public_ingress_remove.py

```diff
@@ -0,0 +1,165 @@
+#
+# This file made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)
+#
+# Ensure that no security groups allow public access to the specified ports.
+# Description: Checks that all security groups block access to the specified ports and removes ip permissions that violate the rule.
+#
+# Trigger Type: Change Triggered
+# Scope of Changes: EC2:SecurityGroup
+# Accepted Parameters: examplePort1, exampleRange1, examplePort2, ...
+# Example Value: 8080, 1-1024, 2375, ...
+
```

Comment:
I'm guessing the lambda execution role needs specific permissions to execute this, what with the revoking of permissions.

Can you call that out here, and add a sample policy?

Reply:
Permissions Required:

AWSConfigRulesExecutionRole (Required for Config service)
AWSLambdaBasicExecutionRole (Required for Lambda service)
Custom EC2 policy			(Allow function to describe SG and revoke IP Permssions Ingress)

{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": [
                "ec2:DescribeSecurityGroups",
                "ec2:RevokeSecurityGroupIngress"
            ],
            "Resource": "*"
        }
    ]
}